tests/test_routes.py

```diff
@@ -70,29 +70,59 @@ def tearDown(self):
     #  H E L P E R   M E T H O D S
     ######################################################################
 
-# Uncomment below when create order API is implemented
-    # def _create_orders(self, count):
-    #     """Factory method to create orders in bulk"""
-    #     orders = []
-    #     for _ in range(count):
-    #         order = OrderFactory()
-    #         print(order)
-    #         resp = self.client.post(BASE_URL, json=order.serialize())
-    #         self.assertEqual(
-    #             resp.status_code,
-    #             status.HTTP_201_CREATED,
-    #             "Could not create test Order",
-    #         )
-    #         new_order = resp.get_json()
-    #         order.id = new_order["id"]
-    #         orders.append(order)
-    #     return orders
+    # Uncomment below when create order API is implemented
+    def _create_orders(self, count):
+        """Factory method to create orders in bulk"""
+        orders = []
+        for _ in range(count):
+            order = OrderFactory()
+            print(order)
+            resp = self.client.post(BASE_URL, json=order.serialize())
+            self.assertEqual(
+                resp.status_code,
+                status.HTTP_201_CREATED,
+                "Could not create test Order",
+            )
+            new_order = resp.get_json()
+            order.id = new_order["id"]
+            orders.append(order)
+        return orders
 
     ######################################################################
     #  P L A C E   T E S T   C A S E S   H E R E
     ######################################################################
 
-    def test_index(self):
-        """It should call the home page"""
-        resp = self.client.get("/")
-        self.assertEqual(resp.status_code, status.HTTP_200_OK)
\ No newline at end of file
+    # def test_index(self):
+    #     """It should call the home page"""
+    #     resp = self.client.get("/")
+    #     self.assertEqual(resp.status_code, status.HTTP_200_OK)
+
+    def test_create_order(self):
+        """It should Create a new Order"""
+        order = OrderFactory()
+        resp = self.client.post(
+            BASE_URL, json=order.serialize(), content_type="application/json"
+        )
+        self.assertEqual(resp.status_code, status.HTTP_201_CREATED)
+
+        # Make sure location header is set
+        # location = resp.headers.get("Location", None)
+        # self.assertIsNotNone(location)
```

Comment:
we commented this out because of the return statement, but if we decide to put it back, could you uncomment this?